### PR TITLE
Add show all pearls command

### PR DESCRIPTION
**Pull request by Protonull**

### CODE REVIEW

**Comment by Convoy20** [comment]
I actually really agree with this

File: src/main/java/com/devotedmc/ExilePearl/command/CmdShowAllPearls.java [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+				.map(this::generatePearlClickable)
+				.collect(Collectors.toCollection(ArrayList::new));
+
+		if (contents.isEmpty()) {
+			final var item = new ItemStack(Material.BARRIER);
+			ItemUtils.setDisplayName(item, ChatColor.RED + "There are currently no pearls.");
+			contents.add(new DecorationStack(item));
+		}
+
+		new MultiPageView(player(), contents, "All Snitches", true).showScreen();
```
Comment (ProgrammerDan) [inline src/main/java/com/devotedmc/ExilePearl/command/CmdShowAllPearls.java]: All Snitches?

Probably "All Pearls" here

File: src/main/java/com/devotedmc/ExilePearl/command/CmdShowAllPearls.java [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+public class CmdShowAllPearls extends PearlCommand {
+
+	private static final DateFormat DATE_FORMAT = new SimpleDateFormat("dd MMM yyyy");
+
+	public CmdShowAllPearls(final ExilePearlApi pearlApi) {
+		super(pearlApi);
+		this.aliases.add("showall");
+		this.setHelpShort("Lists all pearled players.");
+
```
Comment (ProgrammerDan) [inline src/main/java/com/devotedmc/ExilePearl/command/CmdShowAllPearls.java:47]: if this is intentionally available to all players to use, a mirror to this that would be nice is some kind of "checkif" command.

But this looks good!

**Review by ProgrammerDan — CHANGES_REQUESTED** [review]

File: src/main/java/com/devotedmc/ExilePearl/command/CmdShowAllPearls.java (hunk 1 above)
Comment (Protonull) [reply src/main/java/com/devotedmc/ExilePearl/command/CmdShowAllPearls.java]: Oops, been working too much on JukeAlert o.o'

File: src/main/java/com/devotedmc/ExilePearl/command/CmdShowAllPearls.java (hunk 2 above)
Comment (Protonull) [reply src/main/java/com/devotedmc/ExilePearl/command/CmdShowAllPearls.java:47]: Not sure what you mean

Comment (ProgrammerDan) [reply src/main/java/com/devotedmc/ExilePearl/command/CmdShowAllPearls.java:47]: some command to specifically check if a particular player, by name, is pearled.

/checkif Protonull

> Yes, they are pearled, here they are, etc.

Comment (ProgrammerDan) [reply src/main/java/com/devotedmc/ExilePearl/command/CmdShowAllPearls.java:47]: just an idle thought while looking this over, safe to ignore ;)

Comment (Protonull) [reply src/main/java/com/devotedmc/ExilePearl/command/CmdShowAllPearls.java:47]: Aaaaah, okay get it now, definitely sounds like a good idea. I'm not really sure how to do commands with parameters with this particular command system though o.o

Comment (ProgrammerDan) [reply src/main/java/com/devotedmc/ExilePearl/command/CmdShowAllPearls.java:47]: huh, turns out it exists, at least for admins:

https://github.com/CivClassic/ExilePearl/blob/master/src/main/java/com/devotedmc/ExilePearl/command/CmdAdminCheckExiled.java

can tell how often I used this... :D

**Comment by Protonull** [comment]
Just merged in #83 and #86 to fix merge complications

**Pull request merged**